modules/openapi-generator/src/main/resources/python/api_test.mustache

```diff
@@ -28,6 +28,10 @@ class {{#operations}}Test{{classname}}(unittest.TestCase):
         {{{summary}}}  # noqa: E501
 {{/summary}}
         """
+{{#detectAmznApiGatewayExtensions}}
+        response = self.api.{{operationId}}()
+        print(response)
```

Comment:
can you remove print statement? is it needed?

Reply:
It is for testing purposes. When the client uses the detectAmznApiGatewayExtensions flag, it will generate a test file that can be used to invoke the API, and for each method within the test file, the response is received and printed. In order to test, the response would need to be printed to the user, so this is just for ease of the user when testing.